python/examples/notebooks/Katana Tutorial.ipynb

```diff
@@ -64,7 +64,18 @@
    "execution_count": 2,
    "id": "e53dc295",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
The changes in this file seem kinda random. Did you run it manually?

Reply:
besides the `ngreg.diff_ignore` stuff this was all generated by `--nb-force-regen`